frontend/src/bundles/talent-onboarding/components/candidate-profile/profile-second-section/profile-second-section.tsx

```diff
@@ -0,0 +1,229 @@
+    candidateParameters: Record<string, string | number | string[]>;
+    isProfileOpen?: boolean;
+    isFifthStep?: boolean;
+};
+
+const ProfileSecondSection: React.FC<Properties> = ({
+    candidateParameters,
+    isProfileOpen,
+    isFifthStep,
+}) => {
+    const options = [
+        {
+            value: 'Yes',
+            label: 'Yes',
+        },
+        {
+            value: 'No',
+            label: 'No',
+        },
+    ];
+    // fill with real data
+    const { control } = useAppForm<{ hire: '' }>({
+        defaultValues: { hire: '' },
+    });
+
+    return (
+        //  replace with real data
+        <Grid className={styles.profileSecondSection}>
+            <Grid className={styles.candidateInfo}>
+                {isProfileOpen ? (
+                    <Typography variant="h4" className={styles.name}>
+                        {candidateParameters.photo ? (
+                            <img
+                                src={candidateParameters.photo as string}
+                                alt="candidate"
+                            />
+                        ) : (
+                            <Grid className={styles.emptyPhoto}></Grid>
+                        )}
+                        {candidateParameters.fullName}
+                    </Typography>
+                ) : (
+                    <Typography
+                        variant="h4"
+                        className={getValidClassNames(
+                            styles.salary,
+                            styles.salaryTitle,
+                        )}
+                    >
+                        $ {candidateParameters.salaryExpectation}
+                    </Typography>
+                )}
+                <ul className={styles.candidateParamsList}>
+                    {isProfileOpen && (
+                        <li className={styles.candidateParam}>
+                            <img
+                                src={CandidateIcons.SALARY}
+                                className={styles.icon}
+                                alt="salary"
+                            />
+                            <Typography variant="h4" className={styles.salary}>
+                                $ {candidateParameters.salaryExpectation}
+                            </Typography>
+                        </li>
+                    )}
+                    {isProfileOpen && (
+                        <>
+                            <li className={styles.candidateParam}>
+                                <img
+                                    src={CandidateIcons.EMAIL}
+                                    className={styles.icon}
+                                    alt="email"
+                                />
+                                <Typography variant="body1">
+                                    {candidateParameters.email}
+                                </Typography>
+                            </li>
```

Comment:
Looks like we can move this logic to component and use it like separate component

Reply:
done